Tests/sniff-examples/empty_non_variable.php

```diff
@@ -42,5 +42,20 @@
 empty();
 empty( /*comment*/ );
 
+
+// Issue #210 + some extra variants - should all be ok as well.
+empty(${$a . $b });
```

Comment:
Random space after the `$b` here - not sure if you're adhering to some particular standard?